packages/happn-3/package.json

```diff
@@ -80,5 +81,8 @@
     ]
   },
   "protocol": "4",
-  "readmeFilename": "README.md"
+  "readmeFilename": "README.md",
+  "volta": {
```

Comment:
volta looks like it could change the behaviour of command line tools for other folks, are we ok with this being added to package.json?

Reply:
Removed